src/sections/header.liquid

```diff
@@ -5,7 +5,33 @@
     }
   {% endif %}
 </style>
```

Comment:
Should `<style>` tags be below our HTML? I'm leaning on that but it's just a feeling.